### PR TITLE
task(auth): Update new device login subject

### PR DESCRIPTION
**Pull request by dschom**
## Because
- We want to have the subject for new device login emails read 'New sign-in to your Mozilla account' when an account is signed into directly.
- We want to have the subject for new device login emails read 'New sign-in to $clientName' when an account is signed into from an RP

## This pull request

- Adds the ability to customize the subject text for newDeviceLogin emails.
- Follows the same pattern established by lowRecoveryCodes emails.

## Issue that this pull request solves

Closes: FXA-8545

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)
image
image

## Other information (Optional)

A test was _not_ added for this edge case to `tests/local/senders/emails.ts` due to how that test suite is structured;  however, cases were added to storybook.

### CODE REVIEW

**Review by bcolsson — APPROVED** [review]

File: packages/fxa-auth-server/lib/senders/emails/templates/newDeviceLogin/includes.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { GlobalTemplateValues } from '../../../renderer';
+
+export const getIncludes = (clientName: string): GlobalTemplateValues => {
+  const subject =
+    clientName === 'Mozilla account'
```
Comment (vpomerleau) [inline packages/fxa-auth-server/lib/senders/emails/templates/newDeviceLogin/includes.ts:9]: Should this be 'Mozilla' because of:
https://github.com/mozilla/fxa/blob/fbf452be0538b9c9d6216ad7b50205323dda9018/packages/fxa-auth-server/lib/senders/email.js#L816

I was testing locally and the subject line was still coming through as "New sign-in to Mozilla"

Comment (vpomerleau) [reply packages/fxa-auth-server/lib/senders/emails/templates/newDeviceLogin/includes.ts:9]: After changing line 9 above to 'Mozilla', I do see the subject line line changed to "New sign-in to your Mozilla account" (see below in maildev):
image

**Review by vpomerleau — CHANGES_REQUESTED** [review]
Tested locally and confirmed this works after one small change :)

Would this PR also be a good time to change the default sender from 'Firefox accounts' to 'Mozilla' in the config?
https://github.com/mozilla/fxa/blob/fbf452be0538b9c9d6216ad7b50205323dda9018/packages/fxa-auth-server/config/index.ts#L392

**Review by vpomerleau — APPROVED** [review]
Thanks for the updates! LGTM

**Comment by dschom** [comment]
> Thanks for the updates! LGTM

Thanks for catching that config change and the manual testing!
